src/loaders/lottie/tvgLottieBuilder.cpp

```diff
@@ -154,6 +190,16 @@ static bool _updateTransform(LottieTransform* transform, float frameNo, bool aut
         _rotateX(&matrix, transform->rotationEx->x(frameNo, exps));
     }
 
+    auto skewAngle = transform->skewAngle(frameNo, exps);
+    if (fabsf(skewAngle) > 0.01f) {
```

Comment:
If the exceptions are handled in 198, Please consider the most general case.
```
if (skewAngle == 0.0f) 
```
is better obviously.